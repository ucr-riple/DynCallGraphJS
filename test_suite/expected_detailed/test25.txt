{
    'global (test25.js:1:1:19:5)': [
        'anon (test25.js:1:2:19:2)'
    ],
    'anon (test25.js:18:3:18:27)': [
        'show (test25.js:2:3:4:4)'
    ],
    'show (test25.js:3:5:3:11)': [
        'anon (test25.js:6:5:8:6)'
    ]
}