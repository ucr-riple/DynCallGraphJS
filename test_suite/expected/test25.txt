{ '(test25.js:1:1:19:5)':
   [ '(test25.js:1:2:19:2)' ],
  '(test25.js:18:3:18:27)':
   [ '(test25.js:2:3:4:4)' ],
  '(test25.js:3:5:3:11)':
   [ '(test25.js:6:5:8:6)' ] }